temporian/core/data/event.py

```diff
@@ -89,6 +89,10 @@ def __floordiv__(self, other):
     def sampling(self) -> Sampling:
         return self._sampling
 
+    @sampling.setter
+    def sampling(self, sampling: Sampling):
```

Comment:
If this setter is mostly used to create event in several statement (my opposition to the modification of an existing event), would be make more sense not to have a setting, and to ensure the sampling is given in the constructor?